packages/fxa-settings/src/lib/metrics.ts

```diff
@@ -313,6 +313,50 @@ export function logEvents(
   }
 }
 
+/**
+ * Logs an error event
+ * @param viewName
```

Comment:
Q: Is this `@param` accurate? Below it looks like a nested property of the `error` object.

Reply:
Nope. Some auto complete cruft that never got fixed as the method evolved. I'll update accordingly.